ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md

```diff
@@ -0,0 +1,93 @@
+allowing to display a collection of products. 
+
+## Features
+
+* Support for pagination
+* Configurable number of products on one page
+* Client-side loaded prices using GraphQL. Can be disabled in the design dialog.
+* Style System support.
+
+### Use Object
+The Product Collection component uses the `com.adobe.cq.commerce.core.components.models.productcollection.ProductCollection` Sling model as its Use-object.
+
+### Component Policy Configuration Properties
+The following configuration properties are used:
+
+1. `./loadClientPrice` - enables client-side price fetching
+2. `./paginationType` - the pagination type, either `paginationbar` or `loadmorebutton`.
+
+### Edit Dialog Properties
+
+The following properties are written to JCR for this component and are expected to be available as `Resource` properties:
+
+1. `./pageSize` - the number of products shown on one page
+
+## BEM Description
+```
+BLOCK productcollection
+    ELEMENT productcollection__root
+    ELEMENT productcollection__results-count    
+    ELEMENT productcollection__filters
+    ELEMENT productcollection__filters-header
+    ELEMENT productcollection__filters-title
+    ELEMENT productcollection__filters-body
+    ELEMENT productcollection__current-filters
+    ELEMENT productcollection__current-filter
+    ELEMENT productcollection__current-filter-icon
+    ELEMENT productcollection__filter    
+    ELEMENT productcollection__filter-toggler
+    ELEMENT productcollection__filter-header
+    ELEMENT productcollection__filter-icon
+        MOD productcollection__filter-icon--open
+        MOD productcollection__filter-icon--closed    
+    ELEMENT productcollection__filter-items
+    ELEMENT productcollection__filter-item
+    ELEMENT productcollection__filter-title
+    ELEMENT productcollection__items
+    ELEMENT productcollection__item
+        MOD productcollection__item--staged
+    ELEMENT productcollection__item-images
+    ELEMENT productcollection__item-image
+    ELEMENT productcollection__item-image-placeholder
+    ELEMENT productcollection__item-staged
+    ELEMENT productcollection__item-title
+    ELEMENT productcollection__sort
+    ELEMENT productcollection__sort-fields
+    ELEMENT productcollection__sort-title
+    ELEMENT productcollection__sort-keys
+    ELEMENT productcollection__sort-order
+    ELEMENT productcollection__loadmore-button
+    ELEMENT productcollection__loadmore-spinner
+    ELEMENT productcollection__pagination
+    ELEMENT productcollection__pagination-arrow
+        MOD productcollection__pagination-arrow--inactive
+    ELEMENT productcollection__pagination-icon        
+    ELEMENT productcollection__pagination-button
+        MOD productcollection__pagination-button--current
+        MOD productcollection__pagination-button--inactive
+    ELEMENT productcollection__pagination-button-title                
+```
+
+## Information
+* **Vendor**: Adobe
+* **Version**: v1
+* **Compatibility**: AEM as a Cloud Service / AEM 6.4 / 6.5
```

Comment:
I suggest to only use AEM as a Cloud Service & 6.5 for all new components, we don't really test with 6.4 any more and other parts are also only supported on AEM as a Cloud Service & 6.5

Reply:
OK, I've fixed it.